### PR TITLE
#623: solve matching of binary-encoded strings with MySql (via rake rule or initializer setting)

### PR DESCRIPTION
**Pull request by rikettsie**

### CODE REVIEW

**Comment by rikettsie** [comment]
@seuros I cleaned up things a little bit adding a rake rule (or optionally a configuration setting) and updated the README.md consistently.

**Comment by seuros** [comment]
could you squash your commits ?

**Comment by rikettsie** [comment]
Of course @seuros , commits squashed.

**Comment by seuros** [comment]
There is 20 commits in this PR.

**Comment by rikettsie** [comment]
@seuros i finally could squash the commits

File: spec/acts_as_taggable_on/tag_spec.rb [hunk 1]
```diff
@@ -2,8 +2,11 @@
 require 'spec_helper'
 require 'db/migrate/2_add_missing_unique_indices.rb'
 
+
 shared_examples_for 'without unique index' do
-  prepend_before(:all) { AddMissingUniqueIndices.down }
+  prepend_before(:all) do
```
Comment (seuros) [inline spec/acts_as_taggable_on/tag_spec.rb]: why did you change this ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ As such, a _Feature_ would map to either major or minor. A _bug fix_ to a patch.
   * Performance
   * Misc
 
+### [3.4.4 / 2015-02-26](v3.4.4...v3.4.5)
```
Comment (seuros) [inline CHANGELOG.md]: line 7 should be updated.

File: spec/acts_as_taggable_on/tag_spec.rb (hunk 1 above)
Comment (rikettsie) [inline spec/acts_as_taggable_on/tag_spec.rb]: When I implemented the patch as a migration I added the new one there, but then, after changing perspective (as the migration was an "execute", not tracked in the schema) I removed both migration and its call in the tag_spec.rb. Thus that changed. But I can reset the previous line.

File: CHANGELOG.md (hunk 2 above)
Comment (rikettsie) [inline CHANGELOG.md]: ok!

**Comment by seuros** [comment]
for some reason, i can't merge it. can you rebase ?

**Comment by rikettsie** [comment]
@seuros master seems ok now :-)
